openwisp_controller/config/static/config/js/preview.js

```diff
@@ -27,6 +27,14 @@ django.jQuery(function ($) {
             if (name.indexOf('config-0-') === 0) {
                 name = name.replace('config-0-', '');
             }
+            // Systemwide shared objects are identified
+            // with a "null" id in the backend. When submitting
+            // a preview form, Django will raise a validation error 
+            // if the form field corresponding to the object ID 
+            // is not empty or does not contain a valid UUID string.
```

Comment:
I see you wrote this explaination in the commit message which is good, here we can have a shorter version, eg:

```suggestion
            // Avoid sending null which would raise a validation error
```